WalletWasabi.Fluent/Views/Wallets/Send/PrivacyWarningsView.axaml

```diff
@@ -119,6 +119,18 @@
           </Border>
         </DataTemplate>
 
+        <!-- Transaction Known as Yours By Warning -->
+        <DataTemplate DataType="{x:Type model:TransactionKnownAsYoursBy}">
+          <Border Classes="warning info">
+            <DockPanel>
+              <PathIcon />
+              <TextBlock Text="Transaction known as yours by:" Margin="0 0 5 0"
+                         ToolTip.Tip="Those entities know this is your transaction. Coinjoin more or use the label management tool to carefully select who would know that this transaction is yours." />
```

Comment:
```suggestion
                         ToolTip.Tip="This entity knows this is your transaction. Coinjoin more or use the label management tool to carefully select who would know that this transaction is yours." />
```

Reply:
>I think it's alright, I would just change it to singular instead of plural. see suggestion

It is good as it is for now, later we have to make the tooltip dynamic either way (depending if there is a suggestion or not, etc), so it can be fixed there. 

> Should the TransactionKnownAsYoursBy message also be displayed when sending a coin labeled Alice to recipient label Alice?

No, Alice obviously knows about the transaction as the recipient, it would be unnecessary information for the user as they cannot really do anything with it.